src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -1731,6 +1733,9 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
                 llvm_type = list_type;
                 break;
             }
+            case (ASR::ttypeType::Tuple) : {
+                llvm_type = tuple_type;
```

Comment:
First, I am trying to make this simple test work: 
```python
def f():
    t: tuple[f64, f64]
    t = (43.5, 647.75)


f()
```

Reply:
Another simple test,

```
t: tuple[f64, i32, str]
t = (43.5, 0, "a")
```
Making these two work simultaneously would be better.